fingerpost-queueing-lambda/src/handler.ts

```diff
@@ -13,7 +14,7 @@ export const main = async (event: SQSEvent): Promise<SQSBatchResponse> => {
 				const externalId = messageAttributes['Message-Id']?.stringValue;
 				const hasExternalId = externalId && externalId.trim().length > 0;
 				const objectKey = hasExternalId
-					? `${externalId}.json`
+					? formatS3Key(externalId)
```

Comment:
This looks good to me. Should the formatting be fingerpost-specific or apply to all items going via S3? If the latter, should we add this to the poller lambdas too?